docs/integrations/keras_integration.rst

```diff
@@ -0,0 +1,597 @@
+
+I'll walk you through installation and setting up of neptune. In order to install Tensorflow Keras, you can visit this link.
+
+Neptune does not come preinstalled on colab, and also needs to be initialized with an API token before you can log metrics. In the steps below, you're going to create a new Neptune project, install a Neptune client on colab via pip, and also initialize the project with your API token. Follow the steps below to achieve this:
+
+
+Step 1: If you have signed-up, login into your account and create a new project. In the new project settings, add a project name (spam-classifier), copy the initialization code, you can set your project to either public or private, and also add a description. Finally click apply to save the new project.
+
+.. image:: ../_static/images/integrations/keras-integration/kix.9gkqt7221q47.png
+
+.. |signup| raw:: html
+
+    <a href="https://neptune.ai/login" target="_blank">signup</a>
+
+
+.. |colab| raw:: html
+
+    <a href="https://colab.research.google.com" target="_blank">colab</a>
+
+.. |Anaconda| raw:: html
+
+    <a href="https://www.anaconda.com" target="_blank">Anaconda</a>
+
+
+
+Step 2: In your notebook (colab/local) and install neptune client library.
+
+.. code-block:: bash
+
+   pip install neptune-contrib
+
+
+STEP 3: Import neptune, and add your initialization code:
+
+.. code-block:: python3 
+
+   import neptune
+   neptune.init(api_token='ADD-API-TOKEN-HERE',project_qualified_name='risingodegua/spam-classifier')
+
+The code above will initialize your project, and connect your local neptune client to your online account. So that any logs made locally can be synced online.
+
+**Note:** Your API token is a secret, and should be hidden from outsiders. In production environments, it is advisable to add the token to your environment variable. 
+Neptune will automatically pick up the variable from the environment as long as it is called NEPTUNE_API_TOKEN, so you can easily initialize Neptune by running:
+
+.. code-block:: python3
+
+   neptune.init(project_qualified_name='risingodegua/spam-classifier')
+
+
+
+STEP 4: Create your first experiment under your project
+
+.. code-block:: bash
+
+   neptune.create_experiment(name='spam_exploration')
+
+.. image:: ../_static/images/integrations/keras-integration/out1.png
+
+
+Experiments are important for tracking and experimenting on different ideas, and aids reproduction. It is important to always create a new experiment for any new idea you want to try out.
+The |experiment link| provided will open the experiment dashboard where you can see details about your experiments in real time. This link can be shared with collaborators and even the general public if the project has been made public.
+Note: Neptune also tracks hardware metrics to tell you how your machine operates during experimentation. If you're running this experiment locally, you may have to install psutil library first.
```

Comment:
You should mention hardware metrics in the What will you get from this section at the top.